account_statement_completion_compassion/model/statement.py

```diff
@@ -151,16 +155,20 @@ def get_from_amount(self, cr, uid, id, st_line, context=None):
                 [('type', '=', 'in_invoice'), ('state', '=', 'open'),
                  ('amount_total', '=', abs(amount))], context=context)
             res = {}
+            partner_obj = self.pool.get('res.partner')
             if invoice_ids:
                 if len(invoice_ids) == 1:
```

Comment:
Test here seems to be useless. You can simply keep the else part. The loop checking that all invoices will have no side effect on the len = 1 case.